web/pageheader.jspf

```diff
@@ -16,14 +16,13 @@ information: Portions Copyright [yyyy] [name of copyright owner]
 
 CDDL HEADER END
 
-Copyright (c) 2007, 2010, Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2007, 2017, Oracle and/or its affiliates. All rights reserved.
 
 --%><% 
 /* ---------------------- pageheader.jspf start --------------------- */
 {
     cfg = PageConfig.get(request);
 %>
-<a href="<%= request.getContextPath() %>/"><span id="MastheadLogo"></span></a>
```

Comment:
I actually use this logo to return to the homepage all the time. But that's just a matter of habits.